server/src/main/java/org/opensearch/common/settings/Setting.java

```diff
@@ -1009,11 +1009,20 @@ default void validate(T value, Map<Setting<?>, Object> settings, boolean isPrese
     private static class GroupSetting extends Setting<Settings> {
         private final String key;
         private final Consumer<Settings> validator;
+        private Setting<Settings> fallbackSetting;
```

Comment:
Parent class has fallbackSetting implemented so we should change that visibility to `protected` and remove this.